src/selvage/test/flow.clj

```diff
@@ -0,0 +1,199 @@
+(ns selvage.test.flow
+  "clojure.test version of the flow macro"
+   (:require [schema.core :as s]
+             [selvage.core :as core]
+             [clojure.spec.test.alpha :as spec.test]
+             [clojure.test :as t]
+             [selvage.visibility :as vis]
+             [taoensso.timbre :as timbre])
+  (:import [java.io ByteArrayOutputStream PrintStream StringWriter]))
+
+(def ^:dynamic *probe-timeout* 300)
+(def ^:dynamic *probe-sleep-period* 10)
+(def ^:dynamic *verbose* false)
+(def ^:dynamic *world* {})
+(def ^:dynamic *flow* {})
+
+(defn worlds [] (deref core/worlds-atom))
+
+(def ^:dynamic *intermediate-report*)
+
+(defn diff-test-result
+  "Subtract two clojure.test style summary maps."
+  [before after]
+  {:pass  (apply - (map :pass [after before]))
+   :error (apply - (map :error [after before]))
+   :test  (apply - (map :test [after before]))
+   :fail  (apply - (map :fail [after before]))})
+
+(defn report-count []
+  (diff-test-result *intermediate-report* @t/*report-counters*))
+
+(defn update-report-counters [new-reports]
+  (when t/*report-counters*
+    (dosync (commute t/*report-counters*
+                     (fn [current-counters] (merge-with + current-counters new-reports))))))
+
+(defmacro with-report-counters
```

Comment:
Code taken from [`kaocha`](https://github.com/lambdaisland/kaocha/blob/52c23ec391bb3df52f464ee1af6304b37273c8fe/src/kaocha/type.clj#L10).
Thanks @plexus for figuring out how to do this and hope you don't mind me using it here.

Reply:
Not at all :) glad it's being useful!